src/makeInfinityAdder.js

```diff
@@ -7,6 +7,23 @@
 
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  function adder(number) {
+    if (number !== undefined) {
+      sum += number;
+
+      return adder;
+    } else {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    }
```

Comment:
you dont need else here, if first if statement isnt true, next block will run anyway